dashboard/public/pixelation/pixelation.js

```diff
@@ -438,6 +445,18 @@ function setSliders() {
   drawGraph();
 }
 
+function setMinTextValues() {
+  if (heightText.value < 1) {
+    heightText.value = 1;
+  }
+  if (widthText.value < 1) {
+    widthText.value = 1;
+  }
+  if (bitsPerPixelText.value < 1) {
+    bitsPerPixelText.value = 1;
+  }
+}
```

Comment:
I've inverted the comparison so that we use 1 instead of ever returning a non-numerical value. There's no need to cast or parse explicitly from string to int because the comparison operator takes care of that for us and now we deal with `NaN` correctly (`NaN` compared with anything is `false`).